ColinSprows/Fraime
# Journey context

to test:
clear db & local storage
create a new prompt:
- prompt appears in local storage and db
- journey appears in both local storage and db with the correct prompt id
select image to buy in discovery:
- journey updates in both local storage and db with new image id added
